### PR TITLE
expose comment and author as environment variables

### PR DESCRIPTION
**Pull request by v1v**
## Highlights
- Expose GitHub comment and author as environment variables.
  - `GITHUB_COMMENT`
  - `GITHUB_COMMENT_AUTHOR`

## Tests
- https://github.com/super-mario-bros-pipelines/pipeline/pull/1#issuecomment-513401992 caused

image

## Tasks
- [ ] UTs
- [x] UI Tests

### CODE REVIEW

**Comment by aaronjwhiteside** [comment]
Is this intended for comment triggers?

**Comment by v1v** [comment]
yes, it's intended for the comment triggers, I'm still working on the UTs for increasing the code coverage.

**Comment by v1v** [comment]
I've tried to add some UTs but no clue how to test it out :(. Happy to get some assistance if it's required, as the only tests I've have done so far where manual to validate the env variables were created on the fly when the issue commit message matched the regex

File: src/main/java/org/jenkinsci/plugins/pipeline/github/trigger/GitHubParametersAction.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+
+@Restricted(NoExternalUse.class)
+public class GitHubParametersAction extends ParametersAction {
```
Comment (aaronjwhiteside) [inline src/main/java/org/jenkinsci/plugins/pipeline/github/trigger/GitHubParametersAction.java]: Maybe call this GitHubEnvironmentVariablesAction to more closely match what it's doing?

**Comment by aaronjwhiteside** [comment]
So I think environment variables are fine and all, but what would be really cool is exposing the command as a global object, much like the `pullRequest` object. Just a thought, we can leave that for another PR..

@v1v thanks for this!

If you update README.md to document this, I'll merge it soon...

**Comment by tikolakin** [comment]
for some reason it doesn't work for me
I have correct plugging version running
is it supposed to work with both, declarative and scripted pipelines?

**Pull request merged**